scripts/create_bridges.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Service to create bridges
+Before=libvirtd.service
+After=network-online.target 
+After=network.target
+
+[Service]
+WorkingDirectory = /etc/
+ExecStart=/usr/bin/create_bridges
+Type=simple
```

Comment:
I think we want to replace this with `Type=oneshot` here so that we don't have a race condition with libvirt starting VMs before this script finishes.

From the [systemd docs](https://www.freedesktop.org/software/systemd/man/systemd.service.html):
> Behavior of oneshot is similar to simple; however, it is expected that the process has to exit before systemd starts follow-up units.